src/ring_theory/coprime.lean

```diff
@@ -8,6 +8,7 @@ import data.fintype.basic
 import data.int.gcd
 import data.set.pairwise
 import tactic.ring
+import tactic.linarith
```

Comment:
I'm not sure if this file is too basic for this import. Do you think that it makes sense to move the lemma `sq_add_sq_ne_zero` to a different file that deals with linearly ordered rings?

Reply:
I think `sq_add_sq_ne_zero` can move to `coprime/lemmas.lean`.